tests/epyccel/recognised_functions/test_numpy_types.py

```diff
@@ -652,10 +666,10 @@ def test_numpy_float_scalar(language, get_float):
     integer32 = randint(min_int32, max_int32, dtype=np.int32)
     integer64 = randint(min_int64, max_int64, dtype=np.int64)
 
-    fl = uniform(min_float / 2, max_float / 2)
+    fl = uniform(min_float32 / 2, max_float32 / 2)
     fl32 = uniform(min_float32 / 2, max_float32 / 2)
     fl32 = np.float32(fl32)
-    fl64 = uniform(min_float64 / 2, max_float64 / 2)
+    fl64 = uniform(min_float32 / 2, max_float32 / 2)
```

Comment:
why are we using `float32` instead of `float64` in these examples?

Reply:
As above. We are converting to  `float32`, so the `float64` can't contain a float which is too big to be cast without overflow/underflow errors